src/components/projects/A32NX/Download.tsx

```diff
@@ -3,7 +3,7 @@ import { Button } from '../../utils/Button';
 export const Download = ({ expOnHold }: { expOnHold?: boolean }) => {
     const urls = {
         stable: 'https://github.com/flybywiresim/a32nx/releases/download/assets/stable/A32NX-stable.zip',
-        dev: 'https://github.com/flybywiresim/a32nx/releases/download/assets/master/A32NX-master.zip',
+        dev: 'https://github.com/flybywiresim/a32nx/releases/download/assets/master/A32NX-master.7z',
```

Comment:
```suggestion
        dev: 'https://github.com/flybywiresim/aircraft/releases/download/assets/master/A32NX-master.7z',
```
Please update URL for the new repo name

Reply:
Good catch